examples/hazelcast/build.gradle

```diff
@@ -0,0 +1,14 @@
+plugins {
+    id 'java'
+}
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    testImplementation 'org.testcontainers:testcontainers:1.17.5'
```

Comment:
```suggestion
    testImplementation 'org.testcontainers:testcontainers'
```

Reply:
This works without version since we are doing a Gradle composite build with the `examples`.